Enhanced footer (#525)
kitconcept/volto-light-theme: docs/how-to-guides/social-media.md

---
myst:
  html_meta:
    "description": "Social media integration in Volto Light Theme"
    "property=og:description": "Social media integration in Volto Light Theme""
    "property=og:title": "Social media"
    "keywords": "Volto Light Theme, social media"
---

(social-media)=

# Social media

```{versionadded} 6.0.0-alpha.22
```

VLT integrates with the add-on `@plonegovbr/volto-social-media` to provide behaviors for adding social media links and main configuration to your site.

> You can use it to customize your site along with the behaviors in `kitconcept.voltolighttheme` as well if they fit your needs.

Reviewer: ```suggestion
You can use it to customize your site, as well as the behaviors in `kitconcept.voltolighttheme`, if they fit your needs.
```

It has two behaviors:
- `plonegovbr.socialmedia.links`
- `plonegovbr.socialmedia.settings`

> that can be added to any content type (eg. `Plone Site` or `Subsite`) to add the social media links and main configuration to your site.

Reviewer: ```suggestion
You can add these behaviors to any content type, such as `Plone Site` or `Subsite`, to add the social media links and main configuration to your site.
```
